src/stores/AppStore.js

```diff
@@ -531,6 +535,9 @@ export default types
         })
         .then(() => self.setFlags({ isSubmitting: false }));
     }
+    function incrementQueuePosition(number = 1) {
+      self.queuePosition += number;
```

Comment:
If we are just incrementing or decrementing by 1, this should be clamp(self.queuePosition, 0, self.queueTotal) or the numbers could go wild.